### PR TITLE
Rfc changes

### PR DESCRIPTION
**Pull request by PoignardAzur**
My proposed changes so far, separated into commits for easier reading.

### CODE REVIEW

File: text/0000-placement-by-return.md [hunk 1]
```diff
@@ -732,6 +710,40 @@ impl<T> Drop for Filled<T> {
 }
 ```
 
+## Synergy with RFC-1909
+
+This proposal was written to be independent from
+[RFC-1909 (unsized rvalues)](https://github.com/rust-lang/rfcs/blob/master/text/1909-unsized-rvalues.md).
+
+While the two their
```
Comment (notriddle) [inline text/0000-placement-by-return.md]: Dangling sentence is probably not what you want.

**Review by notriddle — COMMENTED** [review]
Just the one minor nit.

File: text/0000-placement-by-return.md (hunk 1 above)
Comment (PoignardAzur) [reply text/0000-placement-by-return.md]: Oops.

**Comment by PoignardAzur** [comment]
Done.

I think this is ready for a PR on the RFC repository.

**Pull request merged**